src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -150,7 +150,7 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
         ]
       ],
       lock_mode: ['COMPLIANCE'],
-      lock_retention_period_days: [0, [CdValidators.number(false), lockDaysValidator]],
+      lock_retention_period_days: [10, [CdValidators.number(false), lockDaysValidator]],
```

Comment:
btw, is this a documented default?

Reply:
This is something I copied from our improvement doc, as I checked last with @cloudbehl  , he mentioned it being default for AWS. cc @cloudbehl 

PS:
0 right now we put as min default retention period which is wrong as it  does not allow bucket creation for 0 retention period. So minimum is 1 numerically which I have updated in html tag.